docker-compose/config/worker-ubuntu22-04.jsonnet

```diff
@@ -31,6 +89,9 @@ local common = import 'common.libsonnet';
       },
     }],
   }],
+  file_pool: {
+    in_memory: {},
```

Comment:
I would strongly recommend using on-disk storage in this case. Or at least add a `TODO` comment here.